src/test/regress/spec/isolation_ref2ref_foreign_keys.spec

```diff
@@ -73,11 +74,19 @@ step "s1-select-table-3"
 
 step "s1-view-locks"
 {
-    SELECT mode, count(*)
-    FROM pg_locks
+    SELECT classid,
+        objid,
+        objsubid,
+        mode,
+        application_name,
+        backend_type,
+        regexp_replace(query, E'[\\n\\r\\u2028]+', ' ', 'g' ) query
+    FROM pg_locks l
+    JOIN pg_stat_activity a
+    ON l.pid = a.pid
```

Comment:
The oid values may change across different PG versions and cause issues. Can we use human readable versions instead?

```suggestion
    SELECT obj.*
        mode,
        application_name,
        backend_type,
        regexp_replace(query, E'[\\n\\r\\u2028]+', ' ', 'g' ) query
    FROM pg_locks l,
         pg_identify_object_as_address(classid, objid, objsubid) obj,
         pg_stat_activity a
    WHERE l.pid = a.pid
```

Reply:
This query already has a WHERE clause, but I guess you see what I mean.

Reply:
they are not oid values, they are shard ids. These columns don't mean what their title suggest for advisory locks.